### PR TITLE
remove all the old tlog and batch referenches from the code

### PR DESCRIPTION
**Pull request by e-orz**
No more old data path. hooray!!!

The monitor of BG has not been refactored yet.

### CODE REVIEW

File: server/cmwell-it/build.sbt [hunk 1]
```diff
@@ -77,7 +77,7 @@ def installCmwell = Def.taskDyn[Array[(String,String)]] {
 }
 
 def launchCmwell(targetDir: File, log: Logger, pe: File): Array[(String,String)] = {
-  log.warn("DO NOT execute Scala REPL nor Cassandra nor Elasticsearch nor Play's Netty server nor cmwell-batch until integration tests are over!")
+  log.warn("DO NOT execute Scala REPL nor Cassandra nor Elasticsearch nor Play's Netty server nor cmwell-bg until integration tests are over!")
```
Comment (hochgi) [inline server/cmwell-it/build.sbt]: if you update the warning, might as well add kafka & zk to the message ;)

File: server/cmwell-it/build.sbt [hunk 2]
```diff
@@ -125,4 +124,5 @@ test in IntegrationTest := Def.task {
 }.tag(Tags.ES,Tags.Cassandra,Tags.Kafka,Tags.Grid).value
 
 //FIXME: if https://github.com/sbt/sbt/issues/3250 is fixed, we should stop manually wiring the tests dependencies like that
-fullTest := (test in IntegrationTest).dependsOn(fullTest in LocalProject("bg"),fullTest in LocalProject("irw"),fullTest in LocalProject("imp"),fullTest in LocalProject("fts"),fullTest in LocalProject("zstore")).value
\ No newline at end of file
+fullTest := (test in IntegrationTest).dependsOn(fullTest in LocalProject("bg"),fullTest in LocalProject("fts"),fullTest in LocalProject("zstore")).value
+fullTest := (test in IntegrationTest).dependsOn(fullTest in LocalProject("bg"),fullTest in LocalProject("fts"),fullTest in LocalProject("zstore")).value
```
Comment (hochgi) [inline server/cmwell-it/build.sbt]: dup line?
also, still need to depend on `irw`...

File: server/cmwell-ws/app/actions/DashBoard.scala [hunk 3]
```diff
@@ -274,19 +271,6 @@ class DashBoard @Inject()(crudServiceFS: CRUDServiceFS)(implicit ec: ExecutionCo
     @volatile private[this] var detail: DetailedHealthData = Map.empty[String,ColorTuple4]
     @volatile private[this] var detailTime = new DateTime(0L)
 
-    def init: Future[(DetailedHealthData,TlogStatus)] = {
```
Comment (hochgi) [inline server/cmwell-ws/app/actions/DashBoard.scala]: again, no context here, but making sure we really don't need the part of `init` that takes care of detailed-health

File: server/cmwell-ws/build.sbt [hunk 4]
```diff
@@ -80,7 +80,8 @@ libraryDependencies ++= {
   dm("org.openrdf.sesame", "sesame-sail-memory"),
   dm("org.openrdf.sesame", "sesame-queryresultio-sparqljson"),
   dm("org.openrdf.sesame", "sesame-queryresultio-text"),
-  dm("org.openrdf.sesame", "sesame-sail-base"))
+  dm("org.openrdf.sesame", "sesame-sail-base"),
+  dm("org.scala-lang", "scala-compiler"))
```
Comment (hochgi) [inline server/cmwell-ws/build.sbt]: srsly...?!?!?
Why are we adding scala compiler here...?

**Comment by GuyGuterman** [comment]
Covered by our tests, and installation (not upgrade) environment without old files.
Moving the issue to Done.

**Pull request merged**